constants.go

```diff
@@ -11,7 +11,8 @@ const (
 	annotationISO8601   = "iso8601"
 	annotationSeperator = ","
 
-	iso8601TimeFormat = "2006-01-02T15:04:05Z"
+	iso8601TimeFormat         = "2006-01-02T15:04:05Z"
+	iso8601ExpandedTimeFormat = "2006-01-02T15:04:05-07:00"
```

Comment:
I'm curious why not just use `time.RFC3339`?

Seems to work fine https://play.golang.org/p/1Rom5WsfCYO

Reply:
Hmmm, I thought that I tried that and it didn't work. But yes that's a much better solution if it does work.